README.md

```diff
@@ -1,14 +1,44 @@
 
 # claim-vat-enrolment-frontend
 
-This is a placeholder README.md for a new repository
+This is a Scala/Play frontend to allow users to claim vat enrolment from HMRC.
 
-### License
+### How to run the service
+
+1. Make sure any dependent services are running using the following service-manager command
+`sm --start CLAIM_VAT_ENROLMENT_ALL -r`
+
+2. Stop the frontend in service manager using
+ `sm --stop CLAIM_VAT_ENROLMENT_FRONTEND`
+ 
+3. Run the frontend locally using
+`./run.sh`
+
+4. Go to the homepage:
+http://localhost:9936/claim-vat-enrolment
 
-This code is open source software licensed under the [Apache 2.0 License]("http://www.apache.org/licenses/LICENSE-2.0.html").
+## End-Points
+### GET /journey/:vrn 
 
-### Starting the service locally
+---
+Retrieves all the journey data that is stored against a specific journeyID
+####Request:
+A valid journeyId must be sent in the URL
 
-To start all the services this depends on run sm --start CLAIM_VAT_ENROLMENT_ALL -r
+#### Response:
+Status: ```SEE_OTHER(301)```             
+
+Example response body:
+```
+{
+"vatRegistrationDate":"2021-01-01",
+"businessPostcode":"AA1 1AA",
+"submittedVatReturns":"Yes",
+"box5Figure":"1000.00",
+"lastVatReturnDate":"January"
+}
+```
```

Comment:
This payload isn't returned to the service on this endpoint. The only response is the redirect to our first page.

Reply:
Does this endpoint return the journey Id to the calling service? if not, should it?